File: services/cadastro-api/2-Application/Cadastro.Application/Dashboard/GetDashboardResumoQuery.cs
using Cadastro.Application.Common.CQRS;

namespace Cadastro.Application.Dashboard;

/// <summary>
/// Query para obter o resumo da dashboard do domínio de Cadastro.
/// Executa COUNTs leves nas tabelas principais — sem paginação.
/// </summary>
public record GetDashboardResumoQuery : IQuery<DashboardResumoResponse>;

File: services/arrecadacao-api/arrecadacao-application/src/main/java/br/com/ecad/arrecadacao/application/queries/GetDashboardResumoQuery.java
package br.com.ecad.arrecadacao.application.queries;

import br.com.ecad.arrecadacao.application.cqrs.Query;
import br.com.ecad.arrecadacao.application.dto.DashboardResumoResponse;

/**
 * Query para obter o resumo da dashboard do domínio de Arrecadação.
 */
public record GetDashboardResumoQuery() implements Query<DashboardResumoResponse> {}
